README.md

```diff
@@ -77,264 +81,298 @@ Community and partner projects:
-
-#### ImageNette
-
-| Model            | Batch Size | Epochs | KNN Test Accuracy |
-|------------------|------------|--------|-------------------|
-| BarlowTwins      |        256 |    800 |             0.850 |
-| BYOL             |        256 |    800 |             0.887 |
-| DCL              |        256 |    800 |             0.864 |
-| DCLW             |        256 |    800 |             0.861 |
-| DINO (Res18)     |        256 |    800 |             0.887 |
-| FastSiam         |        256 |    800 |             0.865 |
-| MAE (ViT-S)      |        256 |    800 |             0.620 |
-| MSN (ViT-S)      |        256 |    800 |             0.833 |
-| Moco             |        256 |    800 |             0.874 |
-| NNCLR            |        256 |    800 |             0.885 |
-| PMSN (ViT-S)     |        256 |    800 |             0.830 |
-| SimCLR           |        256 |    800 |             0.889 |
-| SimMIM (ViT-B32) |        256 |    800 |             0.351 |
-| SimSiam          |        256 |    800 |             0.871 |
-| SwaV             |        256 |    800 |             0.899 |
-| SwaVQueue        |        256 |    800 |             0.898 |
-| SMoG             |        256 |    800 |             0.782 |
-| TiCo             |        256 |    800 |             0.857 |
-| VICReg           |        256 |    800 |             0.843 |
-| VICRegL          |        256 |    800 |             0.781 |
-
-
-#### Cifar10
-
-| Model         | Batch Size | Epochs | KNN Test Accuracy |
-|---------------|------------|--------|-------------------|
-| BarlowTwins   |        512 |    800 |             0.859 |
-| BYOL          |        512 |    800 |             0.910 |
-| DCL           |        512 |    800 |             0.874 |
-| DCLW          |        512 |    800 |             0.871 |
-| DINO          |        512 |    800 |             0.848 |
-| FastSiam      |        512 |    800 |             0.902 |
-| Moco (*)      |        512 |    800 |             0.899 |
-| NNCLR (*)     |        512 |    800 |             0.892 |
-| SimCLR        |        512 |    800 |             0.879 |
-| SimSiam       |        512 |    800 |             0.904 |
-| SwaV          |        512 |    800 |             0.884 |
-| SMoG          |        512 |    800 |             0.800 |
+We provide multi-GPU training examples with distributed gather and synchronized BatchNorm.
+[Have a look at our docs regarding distributed training.](https://docs.lightly.ai/self-supervised-learning/getting_started/distributed_training.html)
+
+
+## Benchmarks
+
+Implemented models and their performance on various datasets. Hyperparameters are not
+tuned for maximum accuracy. Learn more about the benchmarks [here](https://docs.lightly.ai/self-supervised-learning/getting_started/benchmarks.html).
```

Comment:
Should we add here that imagenet benchmarks are coming soon?

Reply:
Let's just add them once they are here :)